sharpy/utils/correct_forces.py

```diff
@@ -0,0 +1,186 @@
+        local_aero_forces (np.ndarray): aerodynamic forces and moments at a grid vertex
+        chi_g (np.ndarray): vector between grid vertex and structural node in inertial frame
+        cbg (np.ndarray): transformation matrix between inertial and body frames of reference
+        force_efficiency (np.ndarray): force efficiency matrix for all structural elements. Its size is ``n_elem x n_node_elem x 2 x 3``
+        moment_efficiency (np.ndarray): moment efficiency matrix for all structural elements. Its size is ``n_elem x n_node_elem x 2 x 3``
+        i_elem (int): element index
+        i_local_node (int): local node index within element
+
+    Returns:
+         np.ndarray: corresponding aerodynamic force at the structural node from the force and moment at a grid vertex
+    """
+
+    n_node = data.structure.num_node
+    n_elem = data.structure.num_elem
+    aero_dict = data.aero.aero_dict
+    new_struct_forces = np.zeros_like(struct_forces)
+
+    # load airfoil efficiency (if it exists); else set to one (to avoid multiple ifs in the loops)
+    airfoil_efficiency = aero_dict['airfoil_efficiency']
+    # force efficiency dimensions [n_elem, n_node_elem, 2, [fx, fy, fz]] - all defined in B frame
+    force_efficiency = np.zeros((n_elem, 3, 2, 3))
+    force_efficiency[:, :, 0, :] = 1.
+    force_efficiency[:, :, :, 1] = airfoil_efficiency[:, :, :, 0]
+    force_efficiency[:, :, :, 2] = airfoil_efficiency[:, :, :, 1]
+
+    # moment efficiency dimensions [n_elem, n_node_elem, 2, [mx, my, mz]] - all defined in B frame
+    moment_efficiency = np.zeros((n_elem, 3, 2, 3))
+    moment_efficiency[:, :, 0, :] = 1.
+    moment_efficiency[:, :, :, 0] = airfoil_efficiency[:, :, :, 2]
+
+    for inode in range(n_node):
+        i_elem, i_local_node = data.structure.node_master_elem[inode]
+        new_struct_forces[inode, :] = struct_forces[inode, :].copy()
+        new_struct_forces[inode, 0:3] *= force_efficiency[i_elem, i_local_node, 0, :] # element wise multiplication
+        new_struct_forces[inode, 0:3] += force_efficiency[i_elem, i_local_node, 1, :]
+        new_struct_forces[inode, 3:6] *= moment_efficiency[i_elem, i_local_node, 0, :]
+        new_struct_forces[inode, 3:6] += moment_efficiency[i_elem, i_local_node, 1, :]
+    return new_struct_forces
+
+# @gen_dict_force_corrections
+def polars(data, aero_kstep, structural_kstep, struct_forces):
+    r"""
+    This function corrects the aerodynamic forces from UVLM based on the airfoil polars provided by the user in the aero.h5 file
+
+    These are the steps needed to correct the forces:
+    * The force coming from UVLM is divided into induced drag (parallel to the incoming flow velocity) and lift (the remaining force).
+    * The angle of attack is computed based on that lift force and the angle of zero lift computed form the airfoil polar and assuming a slope of :math:`2 \pi`
+    * The dreag force is computed based on the angle of attack and the polars provided by the user
```

Comment:
```suggestion
    These are the steps needed to correct the forces:
    
        * The force coming from UVLM is divided into induced drag (parallel to the incoming flow velocity) and lift (the remaining force).
        * The angle of attack is computed based on that lift force and the angle of zero lift computed form the airfoil polar and assuming a slope of :math:`2 \pi`
        * The dreag force is computed based on the angle of attack and the polars provided by the user
```
Added indentation to list (for sphinx not to complain)

Reply:
Thanks, I tried a bunch of different bullets and I couldn't figure out why sphinx didn't show it as a list